feat: Refactor + CI flows (#1)
ReVanced/revanced-manager-play-store-downloader: README.md

      </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://x.com/revancedapp">
      <picture>
         <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/93124920/270180600-7c1b38bf-889b-4d68-bd5e-b9d86f91421a.png">
         <img height="24px" src="https://user-images.githubusercontent.com/93124920/270108715-d80743fa-b330-4809-b1e6-79fbdc60d09c.png" />
      </picture>
   </a>&nbsp;&nbsp;&nbsp;
   <a href="https://www.youtube.com/@ReVanced">
      <picture>
         <source height="24px" media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
         <img height="24px" src="https://user-images.githubusercontent.com/13122796/178032714-c51c7492-0666-44ac-99c2-f003a695ab50.png" />
     </picture>
   </a>
   <br>
   <br>
   Continuing the legacy of Vanced
</p>

# 👋🔌 ReVanced Manager Play Store downloader

![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/ReVanced/revanced-manager-play-store-downloader/release.yml)

> ![GPLv3 License](https://img.shields.io/badge/License-GPL%20v3-yellow.svg)

Reviewer: Make sure that when you click on the label, it redirects you to relevant URL.

Workflow status badge -> workflow status for release.yml

GPLv3 -> License.md

A Google Play Store downloader for ReVanced Manager.

## 🧑‍💻 Usage

- Plugins are managed as Android apps. Download and install the APK file from the releases page.
- After installing, restart ReVanced Manager and enable the plugin in the settings.
- The plugin will now be usable.

When you get prompted, log in to Google with your account. After that, you will be able to download apps from the Play Store.

> [!WARNING]

> > Due to technical limitations, it is only possible to download the latest version of the app. If the ReVanced suggested version differs from the latest, the installation will fail.

Reviewer 2: Docs like this should be put into a /docs folder in the same style as seen in patcher, cli or manager for consistency and scalability. (We have a central documentation repository which will display all of our docs on some webpage eventually so that's why its also functionally relevant)

Author: Agreed to leave this for a separate PR.

## 📚 Everything else

### 📙 Contributing

Thank you for considering contributing to ReVanced Manager Play Store downloader.
You can find the contribution guidelines [here](CONTRIBUTING.md).

> ### 🛠️ Building

Reviewer 2: Same as the usage comment above ^

Author: Agreed to leave this for a separate PR.

> To build ReVanced Manager downloader template, a Java Development Kit (JDK) and Git must be installed.

Reviewer: Build requirements are set to JDK 17, specifically in the step.

Git is not required to build a project, remove it.

```suggestion
To build ReVanced Manager downloader template, a Java Development Kit version 17 (JDK 17).
```

Follow the steps below to build ReVanced Manager downloader template:

> 1. Run `git clone git@github.com:ReVanced/revanced-manager-play-store-downloader.git` to clone the repository
> 2. Run `gradlew assembleRelease` to build the project

Reviewer: Use HTTP instead of SSH [consistency]

```suggestion
1. Run `git clone https://github.com/ReVanced/revanced-manager-play-store-downloader.git` to clone the repository
2. Run `gradlew assembleRelease` to build the project
```

> [!NOTE]
> If the build fails due to authentication, you may need to authenticate to GitHub Packages.
> Create a PAT with the scope `read:packages` [here](https://github.com/settings/tokens/new?scopes=read:packages&description=ReVanced) and add your token to ~/.gradle/gradle.properties.
>
> Example `gradle.properties` file:
>
> ```properties
> gpr.user = user
> gpr.key = key
> ```

> ## 📜 Licence

Reviewer: The word "Licence" is British English spelling, we use primarily American English.

```suggestion
## 📜 License
```
